torchbenchmark/score/configs/v0/config-v0.yaml

```diff
@@ -1,209 +1,209 @@
+    norm: 0.43750642099992093
     weight: 0.008333333333333333
   test_train[attention_is_all_you_need_pytorch-cuda-jit]:
-    norm: 0.3960590936942026
+    norm: 0.43786742414999935
     weight: 0.008333333333333333
   test_train[demucs-cpu-eager]:
-    norm: 0.5079498262377455
+    norm: 0.3546666866000351
     weight: 0.008333333333333333
   test_train[demucs-cpu-jit]:
-    norm: 0.4847838448593393
+    norm: 0.35136531084995115
     weight: 0.008333333333333333
   test_train[demucs-cuda-eager]:
-    norm: 0.13385433438234032
+    norm: 0.16843785500002467
     weight: 0.008333333333333333
   test_train[demucs-cuda-jit]:
-    norm: 0.13377458753529936
+    norm: 0.16847499954997147
     weight: 0.008333333333333333
   test_train[dlrm-cpu-eager]:
-    norm: 0.0017144845532855453
+    norm: 0.0018631149865043692
     weight: 0.05
   test_train[dlrm-cuda-eager]:
-    norm: 0.002243253945507851
+    norm: 0.0034258266816329453
     weight: 0.05
   test_train[fastNLP-cpu-eager]:
-    norm: 0.003055774698101508
+    norm: 0.0023043158927810673
     weight: 0.008333333333333333
   test_train[fastNLP-cpu-jit]:
-    norm: 0.0030127215873820327
+    norm: 0.0022544148208549306
     weight: 0.008333333333333333
   test_train[fastNLP-cuda-eager]:
-    norm: 0.0014317966151229167
+    norm: 0.0017024527120666193
     weight: 0.008333333333333333
   test_train[fastNLP-cuda-jit]:
-    norm: 0.0014074026242601805
+    norm: 0.0016670523807199042
     weight: 0.008333333333333333
   test_train[moco-cuda-eager]:
-    norm: 0.8577832871465944
+    norm: 0.8925068612999667
     weight: 0.016666666666666666
   test_train[moco-cuda-jit]:
-    norm: 0.8417287322808988
+    norm: 0.8923939008500383
     weight: 0.016666666666666666
   test_train[pytorch_mobilenet_v3-cpu-eager]:
-    norm: 0.3875341592472978
+    norm: 0.2377525886000285
     weight: 0.008333333333333333
   test_train[pytorch_mobilenet_v3-cpu-jit]:
-    norm: 0.3767234008410014
+    norm: 0.22350112645001446
     weight: 0.008333333333333333
   test_train[pytorch_mobilenet_v3-cuda-eager]:
-    norm: 0.38706094995141027
+    norm: 0.2387644355000475
     weight: 0.008333333333333333
   test_train[pytorch_mobilenet_v3-cuda-jit]:
-    norm: 0.37686712394934146
+    norm: 0.22566257204994145
     weight: 0.008333333333333333
   test_train[pytorch_struct-cpu-eager]:
-    norm: 1.6595029419171623
+    norm: 6.161046483800055
     weight: 0.008333333333333333
   test_train[pytorch_struct-cpu-jit]:
-    norm: 1.777217530726921
+    norm: 6.181118585249988
     weight: 0.008333333333333333
   test_train[pytorch_struct-cuda-eager]:
-    norm: 0.212453852314502
+    norm: 0.24736640974992952
     weight: 0.008333333333333333
   test_train[pytorch_struct-cuda-jit]:
-    norm: 0.21210659145144745
+    norm: 0.24725276905005558
     weight: 0.008333333333333333
   test_train[yolov3-cuda-eager]:
-    norm: 9.4928129260079
+    norm: 9.692653778850104
     weight: 0.02222222222222222
-target: 1000.0
+target: 1000
```

Comment:
is the change to int type here going to matter? or it's fine bc of how we use it later?

Reply:
Not sure why generate_score_config.py gives different types of result. Fixed